pages/heart.md

```diff
@@ -6,14 +6,14 @@ permalink: /heart
 
 ---
 
-![Image with the 5 phases of the HEART process](/assets/img/heart.png)
+![Image with the 5 phases of the HEART process](/assets/img/heart.png){:height="80%" width="80%"}
 
 ## About
 
 Holistic Enterprise Accessibility Review Technique (HEART) is a proactive, interactive process to ensure government digital services are accessible to everyone.
 
 - [2 Pager](https://docs.google.com/document/d/1rZxWteDvRIJDuc0AN381oDlZhtyZQQZzm5GD3RXbKmY/edit#) - Google Document
-- [Deck](https://docs.google.com/presentation/d/1H6bhM6wqFIds3cXCI3eE0YD65_GmYKzwRviuhs4Jj7s/edit?usp=sharing) - Goolge Sheet
+- [Deck](https://docs.google.com/presentation/d/1H6bhM6wqFIds3cXCI3eE0YD65_GmYKzwRviuhs4Jj7s/edit?usp=sharing) - Goolge Slides
```

Comment:
@mgifford typo in the word google.

Reply:
I don't know Daniel..  Goolge is kinda growing on me..  :)